test/checkEmail.test.js

```diff
@@ -0,0 +1,69 @@
+const chai = require('chai'),
+  sinon = require('sinon'),
+  sinonChai = require('sinon-chai'),
+  checkEmail = require('../middlewares/validators/checkEmail');
+
+const User = require('../lib/modelManagers/usermodel');
+
+chai.use(sinonChai);
+const { expect, should } = chai;
+should();
+
+describe('check if email exist', () => {
+  afterEach(sinon.restore);
+
+  it('should find already existing user email and return conflict', async () => {
```

Comment:
Your case description does not communicate what the test is doing. Reading the description for a test should fully describe the test. Please make your test descriptions a lot more descriptive.

Reply:
fixed